samples/GuestConfiguration/package-samples/README.md

```diff
@@ -0,0 +1,55 @@
+# Azure Policy Guest Configuration samples
+
+These samples are shared to help authors
+to create their own configurations
+and resource modules
+for us in
+[custom Guest Configuration](https://docs.microsoft.com/en-us/azure/governance/policy/how-to/guest-configuration-create) projects.
+
+The community is welcome to contribute changes to these samples,
+or create new samples.
+However, this is not intended to be an Open Source project
+to manage the content used in Azure Policy as built-in content.
+
+## Configurations
+
+[AdministratorsGroupMembers](./configurations/AdministratorsGroupMembers/AdministratorsGroupMembers.mof)<br>
```

Comment:
How do I know what module goes with what .mof?  Or do all of these not need modules?

Reply:
the module requirements are expressed in each mof.  there could be a 1:many between resources and the MOF files where they are used (although I'm not actually sure that is the case yet).
[https://github.com/Azure/azure-policy/blob/23e892b8819e05ec1c960e8b3c928dd46eac6bbe/samples/GuestConfiguration/package-samples/configurations/AdministratorsGroupMembers/AdministratorsGroupMembers.mof#L5](example)